src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -305,7 +305,7 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
                             (store.studies.isComplete && store.sampleLists.isComplete && store.samples.isComplete) &&
                             (<Network genes={store.genes.result!}
                                       profileIds={store.rvQuery.selectedMolecularProfileIds}
-                                      cancerStudyId={store.studies.result[0].studyId}
+                                      cancerStudyId={(store.studies.result.length > 0) ?store.studies.result![0].studyId:""}
```

Comment:
here and everywhere else, ternary spaces should be consistent. just looks more readable.
either `a?b:c` or `a ? b : c`, I think the second one is better. but not `a ?b:c`